hack/go-build.sh

```diff
@@ -24,12 +41,46 @@ TAG=0.7.0
 TAR=${BRANCH}.tar.gz
 URL=${REPO}/archive/${TAR}
 CFLAGS="--redhat --build-tech-preview"
+BUILDER=
+
+while (( $# ))
+do
+case $1 in
+  --image_registry)
+    shift
+    if [[ ! ${1} =~ ^-.* ]] && [[ ! -z "${1}" ]]; then export REGISTRY="${1}"; shift; fi
+  ;;
+  --image_name)
+    shift
+    if [[ ! ${1} =~ ^-.* ]] && [[ ! -z "${1}" ]]; then export IMAGE="${1}"; shift; fi
+  ;;
+  --image_tag)
+    shift
+    if [[ ! ${1} =~ ^-.* ]] && [[ ! -z "${1}" ]]; then export TAG="${1}"; shift; fi
+  ;;
+  --image_builder)
+    shift
+    if [[ ! ${1} =~ ^-.* ]] && [[ ! -z "${1}" ]]; then export BUILDER="${1}"; shift; fi
+  ;;
+  -h|--help)
+    usage
+    exit 0
+  ;;
+  *)
+    echo "Unknown arguments: ${1}"
+    usage
+    exit 1
+  ;;
+esac
+done
 
 setGoModEnv
 go generate ./...
 if [[ -z ${CI} ]]; then
     ./hack/go-test.sh
-    operator-sdk build ${REGISTRY}/${IMAGE}:${TAG}
+    BUILD_PARAMS=""
+    if [[ ! -z ${BUILDER} ]]; then BUILD_PARAMS="${BUILD_PARAMS} --image-builder ${BUILDER}"; fi
+    operator-sdk build ${REGISTRY}/${IMAGE}:${TAG} ${BUILD_PARAMS}
     if [[ ${1} == "rhel" ]]; then
```

Comment:
I think we can drop this, let's take care of productization once we have the Cekit enabled.

Reply:
not sure what would be better, still with cekit or use operator-sdk build.
Maybe have both enabled?

Reply:
I'd rather going for Cekit, like we are doing on downstream, or maybe a `operator-sdk` plugin for cekit :thinking:

Reply:
I don't think cekit will have operator-sdk plugin.
but let's use cekit only for now.

Reply:
what about leave it as it is and update once we migrate to cekit?